specification/vmwarevirtustream/resource-manager/readme.typescript.md

```diff
@@ -0,0 +1,14 @@
+## TypeScript
+
+These settings apply only when `--typescript` is specified on the command line.
+Please also specify `--typescript-sdks-folder=<path to root folder of your azure-sdk-for-js clone>`.
+
+```yaml $(typescript)
+typescript:
+  package-version: 0.2.0
+  azure-arm: true
+  package-name: "@azure/arm-vmwarevirtustream"
+  output-folder: "$(typescript-sdks-folder)/sdk/vmwarevirtustream/arm-vmwarevirtustream"
+  payload-flattening-threshold: 1
```

Comment:
Hello. I'm not sure about this value. In other SDKs, you set it to 2. But we can merge it first and left SDK owners to fix it.

Reply:
add @qiaozha (js sdk owner) for have a double check for this

Reply:
its okay to set this value as 2 for now.  but it's better to have `clear-output-folder: true` .  we can also add this later.